Compile error after PR#26

```
Decide\src\main\LaunchInterceptor.java:83: error: cannot find symbol
            if (points[i - 1].distanceTo(points[i]) > LENGTH1) {
                                                      ^
  symbol:   variable LENGTH1
  location: class LaunchInterceptor
1 error
```
Should be `parameters.LENGTH1`.

--- a/src/main/LaunchInterceptor.java
+++ b/src/main/LaunchInterceptor.java
@@ -78,9 +78,9 @@ public class LaunchInterceptor {
         return true;
     }
 
-    boolean lic0() {
+    public boolean lic0() {
         for (int i = 1; i < numPoints; i++) {
-            if (points[i - 1].distanceTo(points[i]) > LENGTH1) {
+            if (points[i - 1].distanceTo(points[i]) > parameters.LENGTH1) {
                 return true;
             }
         }

